test/unit/files/_util.py

```diff
@@ -42,7 +42,7 @@ def list_root(
 ):
     file_source_pair = file_sources.get_file_source_path(uri)
     file_source = file_source_pair.file_source
-    res = file_source.list("/", recursive=recursive, user_context=user_context)
+    res, count = file_source.list("/", recursive=recursive, user_context=user_context)
```

Comment:
```suggestion
    res, _ = file_source.list("/", recursive=recursive, user_context=user_context)
```
since it's a throwaway variable